src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/queues/v2/common.kt

```diff
@@ -407,6 +407,7 @@ fun refundTransaction(
             .flatMap {
               refundService.requestRedirectRefund(
                 transactionId = transaction.transactionId,
+                touchpoint = transaction.clientId.name,
```

Comment:
this enum actually can be valued with: `CHECKOUT`, `CHECKOUT_CART` and `IO` when CHECKOUT_CART is a specialization for transaction performed with cart on CHECKOUT and should be mapped to CHECKOUT touchpoint for redirect api key retrieval logic
```suggestion
                touchpoint =
                  transaction.clientId.let {
                    when (it) {
                      Transaction.ClientId.CHECKOUT_CART -> Transaction.ClientId.CHECKOUT.name
                      Transaction.ClientId.CHECKOUT,
                      Transaction.ClientId.IO -> it.name
                      else ->
                        throw IllegalArgumentException(
                          "Cannot determine touch point: [$it] for redirect transaction refund")
                    }
                  },
```